devicetypes/fibargroup/fibaro-motion-sensor-zw5.src/fibaro-motion-sensor-zw5.groovy

```diff
@@ -1,315 +1,478 @@
-
-    def cmds = []
-
-    cmds += zwave.wakeUpV2.wakeUpIntervalSet(seconds: 7200, nodeid: zwaveHubNodeId)//FGMS' default wake up interval
-    cmds += zwave.manufacturerSpecificV2.deviceSpecificGet()
-    cmds += zwave.associationV2.associationSet(groupingIdentifier:1, nodeId:[zwaveHubNodeId])
-    cmds += zwave.batteryV1.batteryGet()
-    cmds += zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 1, scale: 0)
-    cmds += zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 3, scale: 1)
-    cmds += zwave.sensorBinaryV2.sensorBinaryGet()
-	cmds += zwave.wakeUpV2.wakeUpNoMoreInformation()
-
-    encapSequence(cmds, 500)
-}
-
-private secure(physicalgraph.zwave.Command cmd) {
-	zwave.securityV1.securityMessageEncapsulation().encapsulate(cmd).format()
-}
-
-private crc16(physicalgraph.zwave.Command cmd) {
-	//zwave.crc16encapV1.crc16Encap().encapsulate(cmd).format()
-    "5601${cmd.format()}0000"
-}
-
-private encapSequence(commands, delay=200) {
-	delayBetween(commands.collect{ encap(it) }, delay)
-}
-
-private encap(physicalgraph.zwave.Command cmd) {
-	def secureClasses = [0x20, 0x30, 0x5A, 0x70, 0x71, 0x84, 0x85, 0x8E, 0x9C]
-
-	//todo: check if secure inclusion was successful
-    //if not do not send security-encapsulated command
-	if (secureClasses.find{ it == cmd.commandClassId }) {
-    	secure(cmd)
-    } else {
-    	crc16(cmd)
-    }
-}
+/**
+ *  Fibaro Motion Sensor ZW5
+ */
+metadata {
+    definition (name: "Fibaro Motion Sensor ZW5", namespace: "FibarGroup", author: "Fibar Group") {
+        capability "Battery"
+        capability "Configuration"
+        capability "Illuminance Measurement"
+        capability "Motion Sensor"
+        capability "Sensor"
+        capability "Tamper Alert"
+        capability "Temperature Measurement"
+        capability "Health Check"
+        capability "Three Axis"
+
+        command "resetMotionTile"
+        command "forceSync"
+
+        fingerprint mfr: "010F", prod: "0801", model: "2001"
+        fingerprint deviceId: "0x0701", inClusters: "0x5E, 0x20, 0x86, 0x72, 0x5A, 0x59, 0x85, 0x73, 0x84, 0x80, 0x71, 0x56, 0x70, 0x31, 0x8E, 0x22, 0x30, 0x9C, 0x98, 0x7A"
+        fingerprint deviceId: "0x0701", inClusters: "0x5E, 0x20, 0x86, 0x72, 0x5A, 0x59, 0x85, 0x73, 0x84, 0x80, 0x71, 0x56, 0x70, 0x31, 0x8E, 0x22, 0x30, 0x9C, 0x7A"
+    }
+
+    tiles(scale: 2) {
+        multiAttributeTile(name:"FGMS", type:"lighting", width:6, height:4) {
+            tileAttribute("device.motion", key:"PRIMARY_CONTROL") {
+                attributeState("inactive", label:"no motion", icon:"https://s3-eu-west-1.amazonaws.com/fibaro-smartthings/motionSensor/motion0.png", backgroundColor:"#ffffff")
+                attributeState("active", label:"motion", icon:"https://s3-eu-west-1.amazonaws.com/fibaro-smartthings/motionSensor/motion1.png", backgroundColor:"#00a0dc")
+            }
+            tileAttribute("device.multiStatus", key:"SECONDARY_CONTROL") {

```

Comment:
custom attributes should be declared up at the top using the `attribute` function